src/adaptors/http1/http1_server.c

```diff
@@ -70,6 +70,7 @@ typedef struct _server_request_t {
     bool            request_settled; // set by adaptor
     bool            request_acked;   // true if dispo sent to core
     bool            headers_encoded; // True when header encode done
+    bool            response_settled;
```

Comment:
Sorry - late night but for the life of me I don't see where this flag is set!  I'd assume it would be set in qdr_http1_server_core_delivery_update when the outcome for the response arrives...

Reply:
Agreed I have removed that.